cmd/okctl/handlers/application_test.go

```diff
@@ -0,0 +1,55 @@
+package handlers
```

Comment:
No tests to ensure that the `apply application` path through HandleApplication will execute correctly. The `opts.Purge` and `preConfirmed` variables are important for correct behavior

Reply:
I was unable to test preConfirmed due to our prompt lib having a less than ideal way of configuring IO, but purge flag tests are added

Reply:
@haavardeide